cli/lib/soupify/run-entrypoint-file.ts

```
@@ -17,44 +17,49 @@ export const runEntrypointFile = async (
   }: { tmpEntrypointPath: string; tmpOutputPath: string },
   ctx: Pick<AppContext, "runtime" | "params">,
 ) => {
-  debug(`using runtime ${ctx.runtime}`)
-  const processCmdPart1 =
-    ctx.runtime === "node"
-      ? $`npx tsx ${tmpEntrypointPath}`
-      : $`bun ${tmpEntrypointPath}`
-
-  debug(`starting process....`)
-  const processResult = await processCmdPart1
-    .stdout(debug.enabled ? "inheritPiped" : "piped")
-    .stderr(debug.enabled ? "inheritPiped" : "piped")
-    .noThrow()
+  try {
+    debug(`using runtime ${ctx.runtime}`)
+    const processCmdPart1 =
+      ctx.runtime === "node"
+        ? $`npx tsx ${tmpEntrypointPath}`
+        : $`bun ${tmpEntrypointPath}`
+    debug(`starting process....`)
 
-  const rawSoup = await readFile(tmpOutputPath, "utf-8")
-  const errText = processResult.stderr
+    try {
+      const processResult = await processCmdPart1
+        .stderr(debug.enabled ? "inheritPiped" : "piped")
+        .stdout(debug.enabled ? "inheritPiped" : "piped")
+    } catch (e) {
+      console.log(e)
```

hmm we should probably revert this part because it silences errors

---

*continuing after errors